### PR TITLE
Bug 1507225 - Add basic support for launching the GeckoView Example app

### PR DESCRIPTION
**Pull request by staktrace**

### CODE REVIEW

File: mozregression/launchers.py [hunk 1]
```diff
@@ -388,6 +388,72 @@ def get_app_info(self):
...
+
+    def _stop(self):
+        self.adb.stop_application(self.package_name)
+        if self.adb.exists(self.remote_profile):
+            self.adb.rm(self.remote_profile, recursive=True)
+
+    def get_app_info(self):
+        return self.app_info
+
+
```
Comment (wlach) [inline mozregression/launchers.py]: It looks like this is 99% the same as the FennecLauncher class elsewhere in the file. Could you create a base class called `AndroidLauncher` and have both this and FennecLauncher inherit from it?

File: mozregression/fetch_build_info.py [hunk 2]
```diff
@@ -270,6 +271,7 @@ def find_build_info(self, date, fetch_txt_info=True, max_workers=2):
             for thread in threads:
                 while thread.is_alive():
                     thread.join(0.1)
+            LOG.debug("got valid_builds %s" % valid_builds)
```
Comment (wlach) [inline mozregression/fetch_build_info.py]: Could you remove these debugging logs as well?

Comment (staktrace) [reply mozregression/fetch_build_info.py]: Is there rationale for when debug logs are ok vs not? I found these logs useful when debugging the code, so I figured they might be useful to others too.

File: mozregression/launchers.py (hunk 1 above)
Comment (staktrace) [reply mozregression/launchers.py]: Yup, will do

File: mozregression/fetch_build_info.py (hunk 2 above)
Comment (wlach) [reply mozregression/fetch_build_info.py]: Not really. Actually, on second thought, let's leave these in. They shouldn't be visible under normal circumstances.

**Comment by wlach** [comment]
Thank you! I will do a new release with these changes soon

**Pull request merged**